js/redirects.js

```diff
@@ -26,6 +38,19 @@ $( document ).ready(function() {
         "to": "http://kubernetes.io/docs/whatisk8s/"
     }];
 
+    for (var i = 0; i < apiv1.length; i++) {
+        if (forwardingURL.indexOf(apiv1[i].from) > -1) {
+            var re = new RegExp(apiv1[i].pattern, 'g');
+            var matchary = re.exec(forwardingURL);
+            var newURL = apiv1[i].to;
+            if (matchary !== null) {
+                newURL += apiv1[i].postfix.replace("<token>", matchary[1]);
+            }
+            notHere = true;
+            window.location.replace(newURL);
+        }
+    }
+
```

Comment:
This change seems to do what we want overall, but it's all happening after the page loads (so the 404 is displayed briefly). I think I've seen this before as well. It's fine for now, but maybe we should have an issue to fix that in the future, by redirecting before the page load event.